### PR TITLE
Connectivity - Setting up a Relay endpoint

### PR DESCRIPTION
**Pull request by daviddias**

### CODE REVIEW

**Comment by daviddias** [comment]
Preview location https://deploy-preview-206--filecoin-docs.netlify.app/mine/connectivity/#setting-up-a-relay-endpoint

File: docs/mine/connectivity.md [hunk 1]
```diff
@@ -86,3 +82,58 @@ Below is an example `~/.lotusminer/config.toml` configuration file in which the
...
+Note: Remember that libp2p (the underlaying network stack of the Filecoin miner) will be listening on multiple addresses simultaneously. What this means is that adding a Relay endpoint is not a tradeoff but an advantage, as it will be used for last resort when direct connectivity can't be achieved.
+
+### libp2p Relay
+
+The [libp2p Circuit Relay (docs here)](https://docs.libp2p.io/concepts/circuit-relay/) is just a standard libp2p node that offers a service to any other Relay node to route their traffic through it. With it, you can deploy a libp2p Circuit Relay in a machine with a Public IP address (e.g. a standard Cloud Provider) and by adding a libp2p relay multiaddr to your miner node, you will instruct the miner to route all of its traffic through the libp2p Relay.
+
+You can learn:
+- How it works [here](https://docs.libp2p.io/concepts/circuit-relay)
+- How to write a simple program that does the relay [here](https://github.com/libp2p/go-libp2p-examples/blob/master/relay/main.go)
+
```
Comment (daviddias) [inline docs/mine/connectivity.md:101]: @gmas, @thattommyhall I was asking for a libp2p Relay deployment example so that I could add it here.

**Comment by mishmosh** [comment]
Thanks for this! Should most miners set multiaddrs AND relay endpoints?

**Comment by momack2** [comment]
@mishmosh - AFAIK using a relay is an alternative way to set a public IP address. AKA - you don't need to locally have a public IP if you are relaying traffic to your node on a private network from a public port/IP.

**Comment by mishmosh** [comment]
In that case, this new section should be presented together with the prior one on setting a public IP.

File: docs/mine/connectivity.md (hunk 1 above)
Comment (momack2) [reply docs/mine/connectivity.md:101]: IIRC Juan suggested we shouldn't provide an existing deployment

**Comment by mishmosh** [comment]
> In that case, this new section should be presented together with the prior one on setting a public IP.

For time, I added a commit with this change + spelling nits, and plan to merge once tests pass. Happy to discuss or followup in a separate PR.

**Pull request merged**